docs/pages/router/basics/notation.mdx

```diff
@@ -0,0 +1,76 @@
+
+### Simple names / no notation
+
+<FileTree files={[['app/home.tsx'], ['app/feed/favorites.tsx']]} />
+
+Regular folder and file names without any notation signify _static routes_. Their URL matches exactly as they appear in your file tree. So, a file named **favorites.tsx** inside the **feed** folder will have a URL of `/feed/favorites`.
+
+### Square brackets
+
+<FileTree files={[['app/[userName].tsx'], ['app/products/[productId]/index.tsx']]} />
+
+If you see square brackets in a folder or file name, you are looking at a _dynamic route_. The name of the route includes a parameter that can be used when rendering the page. The parameter could be either in a folder name or a file name. For example, a file named **[userName].tsx** will match `/evanbacon`, `/expo`, or another username. Then you can access that parameter with the `useLocalSearchParams` hook inside the page, using that to load the data for that specific user.
+
+### Parentheses
+
+<FileTree files={[['app/(tabs)/index.tsx'], ['app/(tabs)/settings.tsx']]} />
+
+A folder with its name surrounded in parenthesis indicates a _route group_. These folders are useful for grouping routes together without affecting the URL. For example, a file named **(tabs)/settings.tsx** will have `/settings` for its URL, even though it is not directly in the **app** directory.
+
+Route groups could be useful for simple organization purposes, but often becomes more important for defining complex relationships between routes.
+
+### index.tsx files
+
+<FileTree files={[['app/(tabs)/index.tsx'], ['app/profile/index.tsx']]} />
+
+Just like on the web, an **index.tsx** file indicates the default route for a folder. For example, a file named **profile/index.tsx** will match `/profile`. A file named **(tabs)/index.tsx** will match `/`, effectively becoming the default route for your entire app.
+
+### \_layout.tsx files
+
+<FileTree files={[['app/\_layout.tsx'], ['app/(tabs)/\_layout.tsx'], ['app/feed/\_layout.tsx']]} />
+
+**\_layout.tsx** files are special files that are not pages themselves, but define how groups of routes inside a folder relate to each other. If a folder of routes are arranged as a stack or tabs, the layout route is where you would define that relationship by using a stack navigator or tab navigator component.
+
+Layout routes are rendered prior to the actual page routes inside their folder. This means that the **\_layout.tsx** directly inside the app folder is rendered before anything else in the app, and is where you would put initialization code that may have previously gone inside an **App.js** file.
+
+### Plus sign
+
+<FileTree files={[['app/+not-found.tsx'], ['app/+native-intent.tsx']]} />
+
+Routes starting with a “+” have special significance to Expo Router. One example is `+not-found`, which is a catches any requests that don't match a route in your app. You’ll learn more about these as you need them.
```

Comment:
```suggestion
Routes starting with a `+` have special significance to Expo Router. One example is `+not-found`, which catches any requests that don't match a route in your app.
```

I don't think there's any significance here for the last line unless we link to something.